zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -303,11 +305,17 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
 
   $scope.runParagraph = function(data) {
     websocketMsgSrv.runParagraph($scope.paragraph.id, $scope.paragraph.title,
-                                 data, $scope.paragraph.config, $scope.paragraph.settings.params);
+                                 data, $scope.paragraph.config, $scope.paragraph.settings.params,
+                                 $scope.paragraph.requestParam);
     $scope.originalText = angular.copy(data);
     $scope.dirtyText = undefined;
   };
 
+  $scope.runParagraphAndMoveFocusToNext = function(data) {
```

Comment:
2) Overall, this function and `$scope.paragraph.requestParam` are not needed.
By passing a parameter to the `$scope.run(boolean)` specified in 1), you will be able to set the `movefocusToNextParagraph`.

Example:
```
$scope.run = function(moveFocus) {
....
websocketMsgSrv.runParagraph($scope.paragraph.id, $scope.paragraph.title,
                                  data, $scope.paragraph.config, $scope.paragraph.settings.params);
                                  data, $scope.paragraph.config, $scope.paragraph.settings.params,
                                 {moveFocusToNextParagraph: moveFocus}
);
```